tests/prog/test_triple_refinement.py

```diff
@@ -5,7 +5,7 @@
 @mark.asyncio
 async def test_triple_refinement(test_preprocessing):
     from honeybadgermpc.mpc import TaskProgramRunner
-    from honeybadgermpc.triple_refinement import refine_triples
+    import prog.triple_refinement as triple_refinement
```

Comment:
oops,  I had a typo I meant `progs` and not `prog`, can you please fix this?

Reply:
Why do you need to import the whole file?  `from prog.triple_refinement import triple_refinement` should also work, right?

Reply:
I don't need to import everything-- fixed both of these.